decidim-core/app/commands/decidim/search.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module Decidim
+  # A command that will act as a search service, with all the business logic for performing searches.
+  class Search < Rectify::Command
+    ACCEPTED_FILTERS = [:resource_type, :decidim_scope_id].freeze
+
+    attr_reader :term, :results
+
+    # Public: Initializes the command.
+    #
+    # @param term: The term to search for.
+    def initialize(term, organization, filters = nil)
+      @term = term
+      @organization = organization
+      @filters = filters || {}
```

Comment:
If you set `filters = {}` in the method definition, you'll avoid this `|| {}` part